requirements/install.txt

```diff
@@ -0,0 +1,8 @@
+# these are dependencies required to use the package
+# ought to mirror 'instal_requires' under 'options' in config.cfg
+astropy >= 3.1
+colorama >= 0.3
+matplotlib >= 3.2
+numpy >= 1.14
```

Comment:
```suggestion
numpy >= 1.16
```

Reply:
Does it, now? That's likely a sign of units issues going away!